src/DB/Query/ShippingRateQuery.php

```diff
@@ -67,4 +67,27 @@ function ( $row ) {
 			$this->results
 		);
 	}
+
+	/**
+	 * Get all shipping rates.
```

Comment:
This description doesn't really define why it's needed over just calling `get_results`. As far as I can see the core part of this function is to format the shipping rates in a specific way and index them by country. If so then lets clarify that in this docblock.

I suppose it's a direct copy of `get_all_shipping_times`. But since we are adding it new, lets use the opportunity to clarify it's purpose.